src/librustdoc/html/static/settings.css

```diff
@@ -30,6 +29,45 @@
 	display: none;
 }
 
+.select-wrapper {
+	float: right;
+
+	position: relative;
```

Comment:
I'm just wondering: is there any particular reason for these empty lines?

Reply:
Oh my bad, that's just a habit I have. I tend to try and separate groups of rules that kind of go together (e.g. position followed by top/right/left/bottom). I'll remove them 👍